samples/teams-sso-node/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "node-teams-sso",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "cpx ../../packages/mgt/dist/bundle/* ./client/mgt && nodemon server.ts",
```

Comment:
Add `"` around params

Reply:
@nmetulev I saw that what you did made it work, but have a look at this: https://github.com/microsoftgraph/microsoft-graph-toolkit/blob/b3a50b93885108e3fd63231a2410504045236cf7/packages/mgt/package.json#L35

Reply:
Good point, not sure why it was causing issues for me 🤷‍♂️ - the quotes won't hurt though either way - I think we should still add them.

Reply:
Yes, I did add the quotes now and tested that it works :) I'll update the PR soon.
Just gonna change the sample documetation a little